custom_components/tahoma/cover.py

```diff
@@ -31,4 +31,11 @@ async def async_setup_entry(
         if device.ui_class != UIClass.AWNING
     ]
 
+    entities += [
+        VerticalCover(device.device_url, data.coordinator, low_speed=True)
+        for device in data.platforms[Platform.COVER]
+        if device.ui_class != UIClass.AWNING
+        and OverkizCommand.SET_CLOSURE_AND_LINEAR_SPEED in device.definition.commands
+    ]
+
```

Comment:
Does this overrides the entities previously added for the ones having the low speed command?

Reply:
Nope, an other one is added. So the user will have two cover entities for the same device.